lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -112,11 +112,15 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
    * @return {string}
    */
   _getScoringCalculatorHref(auditRefs) {
-    const v5andv6metrics = auditRefs.filter(audit => audit.group === 'metrics');
-    const fci = auditRefs.find(audit => audit.id === 'first-cpu-idle');
-    const fmp = auditRefs.find(audit => audit.id === 'first-meaningful-paint');
-    if (fci) v5andv6metrics.push(fci);
-    if (fmp) v5andv6metrics.push(fmp);
+    const metrics = auditRefs.filter(audit => audit.acronym);
```

Comment:
this feels a little less clear and AFAICT no audits changed groups in this PR. why the switch to acronym?

Reply:
yeah fair.  i reverted the code changes and left a todo.

the code's a little awkward as acronym was just added, but we want to support old LHRs that don't have it. (and we want to get FMP/FCPUI numbers too, despite them being hidden).

anyway, should be better?